docs/core/resilience/snippets/resilience/Program.cs

```diff
@@ -36,11 +37,11 @@
 // </pipeline>
 
 // <execute>
-await pipeline.ExecuteAsync(static async cancellationToken =>
+await pipeline.ExecuteAsync(static cancellationToken =>
 {
     // Code that could potentially fail.
 
-    await ValueTask.CompletedTask;
+    return default; // ValueTask.CompletedTask;
```

Comment:
Why change this?

Reply:
return default; is sufficient to compile and does not require an async state machine.

Reply:
@IEvangelist Can you accept my rationale or shall I rollback this change to your original?